Expensify/App
# Focus emoji picker search input

### Details
Following-up on https://github.com/Expensify/Expensify.cash/pull/1991#event-4602076155. During testing, we found a minor issue that when the `EmojiPickerMenu` is first opened, neither the `ReportActionCompose` nor the emoji search input are focused. We decided that because that was already a big PR it would be easier to handle the focus issue in a follow-up PR.

**Note:** I also found an issue with the existing implementation where the search input was displayed on mobile web, which we don't want for the same reasons we don't want it to display on mobile (i.e: the native keyboard competes with the emoji picker.

### Fixed Issues
n/a.

### Tests / QA Steps (Web/Desktop)
1. Click on the emoji icon in the `ReportActionCompose` to open the emoji picker.
1. Verify that the emoji picker search input is focused, and that you can immediately type to search emojis.
1. Click an emoji and verify that it populates the `ReportActionCompose` (make sure it doesn't take two clicks)
1. Close the emoji picker.
1. Verify that the `ReportActionCompose` has focus again.

### Tests / QA Steps (iOS/Android/mWeb)
Just make sure that the emoji picker still works like normal. This PR should have no other effect.

### Tested On


### Screenshots

#### Web
![image](https://user-images.githubusercontent.com/47436092/114953090-7edad980-9e0c-11eb-8b0e-3dd829f665c2.png)

#### Mobile Web
![image](https://user-images.githubusercontent.com/47436092/114956215-45599c80-9e13-11eb-946c-c6f57bb969f8.png)

#### Desktop
![image](https://user-images.githubusercontent.com/47436092/115055786-cd34ba80-9e96-11eb-8244-acbc6635e018.png)

#### iOS
![image](https://user-images.githubusercontent.com/47436092/114956915-bc436500-9e14-11eb-8abb-17b323d298a2.png)

#### Android
![image](https://user-images.githubusercontent.com/47436092/115057548-0bcb7480-9e99-11eb-90a2-74c386819dd3.png)
